easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.10'
+
+homepage = 'http://www.ece.uvic.ca/~frodo/jasper/'
+description = """The JasPer Project is an open-source initiative to provide a free
+ software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard."""
+
+separate_build_dir = True
```

Comment:
Ah, sorry, of course, I was mixing this up with `build_in_installdir` for some reason...

Ignore my rant, but please do move it down as requested.